nix/configs/production/worker.json

```diff
@@ -5,7 +5,7 @@
   "taskcluster_scope_prefix": "project:releng:ship-it:",
   "ship_it_instances": {
     "project:releng:ship-it:server:staging": {
-        "api_root_v2": "https://shipit-api.mozilla-releng.net",
+        "api_root_v2": "http://shipitapi-prod-shipitapi-app-1",
```

Comment:
maybe it would be nice to add a comment why is this pointing to http.

Reply:
ignore above. this is json. not for future and why we should never use json for config :)

Reply:
JSON won't let you do that, and we also verify the JSON against the schema, so no "_comment" fields or something  :/